packages/react-examples/src/react-charting/PieChart/PieChart.Basic.Example.tsx

```diff
@@ -1,26 +1,66 @@
 import * as React from 'react';
 import { PieChart, IPieChartProps } from '@fluentui/react-charting';
-import { DefaultPalette } from '@fluentui/react/lib/Styling';
+import { Stack, StackItem } from '@fluentui/react';
 
-export class PieChartBasicExample extends React.Component<IPieChartProps, {}> {
+export class PieChartBasicExample extends React.Component<IPieChartProps, { width: number; height: number }> {
   constructor(props: IPieChartProps) {
     super(props);
+    this.state = {
+      height: 350,
+      width: 600,
+    };
   }
 
   public render(): JSX.Element {
     const points = [
-      { y: 50, x: 'A' },
-      { y: 25, x: 'B' },
-      { y: 25, x: 'C' },
+      { y: 50, x: 'ABCD' },
+      { y: 25, x: 'EFGH' },
+      { y: 25, x: 'IJKL' },
     ];
-    const colors = [DefaultPalette.red, DefaultPalette.blue, DefaultPalette.green];
     return (
-      <PieChart
-        culture={window.navigator.language}
-        data={points}
-        chartTitle="Pie Chart basic example"
-        colors={colors}
-      />
+      <React.Fragment>
```

Comment:
I think pie supports the case when no of colors is less than no of datapoints.

Reply:
Yes currently the user can provide any no of colors - less, equal, more or skip the prop. It'll be supported. (Removed this to demonstrate the default color behavior in the examples)

Then furthermore user can provide either the css colors (hex value) or the color from DataVizPalette.

Reply:
Yes this is fixed now, if there's less no of colors, it will repeat those colors

Reply:
In this example it's not passed, to show that it's optional now, which it was not possible earlier